swan-lake/learn-the-platform/ballerina-persist/overview.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: ballerina-persist-documentation-left-nav-pages-swanlake
+title: Bal persist overview
+description: The section gives an overview of the `bal persist` feature.
+keywords: ballerina, programming language, ballerina packages, persist, persist model, persist cli, persist client api
+permalink: /learn/ballerina-persist/persist-overview/
```

Comment:
```suggestion
permalink: /learn/ballerina-persist/persist-overview/
```
Currently, the link on the Learn landing page for this page is broken. Also, the text is added as `Overview` and it should be changed to `Persist overview`.